### PR TITLE
Add notes on facilitation and TOC

### LINKED ISSUE #17: Add notes for facilitators
(from my email:

- come having thought about the texts
- write up 2-4 "themes" or questions you are interested in from the readings
- keep time (and gently wrap is up toward next week)
  - can throw to me or matt p. in terms of why commons/set up for next month if you want
- ask for someone to take notes...

pad we use: https://hackmd.io/oEcuKALCTi-PbawLmT_Ixw#)

### PR DESCRIPTION
**Pull request by dcwalk**
Fixes #17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,6 +107,21 @@ What is right matters, why go to justice (rather than...), what is fair is disce
...
+## Facilitation
+
+Here are some guidelines about preparation in order to facilitate a session:
+
+- Before the session: 
+  - Have read and thought about the texts
+  - Identify and write 2-4 "themes" or questions you are interested in from the readings
+  - We use the same [shared notepad](https://hackmd.io/oEcuKALCTi-PbawLmT_Ixw#): https://hackmd.io/oEcuKALCTi-PbawLmT_Ixw#, which has a template for notes
+- During the session:
+  - Arrive on time (call should auto record as the first person enters)
```
Comment (Mr0grog) [inline README.md]: Given today’s experience, maybe recording be another bullet:

> - Ensure the call is recorded (it should auto-record as the first person enters, but always make sure someone presses the record button if not)

**Comment by dcwalk** [comment]
@meiqimichelle maybe you could also take a look given that you most recently were in the position of new facilitator? 😄  (also invite to the org sent! join usssss.....)

File: README.md (hunk 1 above)
Comment (dcwalk) [reply README.md]: thanks for the review and suggestion, I've incorporated!

**Comment by dcwalk** [comment]
@titaniumbones could you take a quick look at this today/tomorrow morning?
I'm going to self-merge shortly if not to ensure we are prepared for next week

**Pull request merged**